### PR TITLE
Add extra attributes to images on Aztec.

### LINKED ISSUE #7537: Uploaded images missing attributes
Originally reported by @elibud in https://github.com/wordpress-mobile/AztecEditor-iOS/issues/603

### Expected behavior
When I create a post and upload an image to it, I expect that image to be displayed on the post list and on reader, both on the apps and on the web.

### Actual behavior
The post doesn't display an image in the following places:
- Reader iOS
- Post List iOS
- Post List Calypso

The reason behind this seems to be that the uploaded image is missing the `width` attribute. As a reference here's how an image uploaded via the Hybrid editor looks like:
`<a href="https://elibudandpets.files.wordpress.com/2017/07/img_6963.jpg"></a>`
And here's one uploaded with Aztec:
``

### Steps to reproduce the behavior
Create a new post, upload an image, check the post on Reader and the site's Post List.

##### Tested on IPhone 7, iOS 10.3, Aztec beta 6

### PR DESCRIPTION
**Pull request by SergioEstevao**
Fixes #7537 

To test:
 - Create a new post
 - Add an image to it 
 - Wait for upload to finish
 - Switch to HTML mode
 - See if width, height and wp-image-XX properties are applied to img tag.

Needs review: @elibud

### CODE REVIEW

File: WordPress/Classes/ViewRelated/Aztec/Extensions/ImageAttachment+WordPress.swift [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+            var attributes = [String]()
+            if let classAttribute = extraAttributes["class"] {
+                attributes = classAttribute.components(separatedBy: " ")
+            }
+            attributes = attributes.filter({ (value) -> Bool in
+                !value.hasPrefix("wp-image-")
+            })
+            if let nonNilValue = newValue {
+                attributes.append("wp-image-\(nonNilValue)")
+
```
Comment (elibud) [inline WordPress/Classes/ViewRelated/Aztec/Extensions/ImageAttachment+WordPress.swift]: Extra empty line here.

**Comment by elibud** [comment]
I have good news and bad news:

Good news first: the original bug is fixed! I created a new post, verified it on reader, post list and post list Calypso, all of them look good. I also looked at the HTML and initially it looked good to me.

Bad news: after publishing, the class attributed had the value wp-image repeated several times:
image uploaded from ios 1

A little bit more testing revealed that the attribute is added one time for every switch between visual and html mode.

I'm leaving the proper code review to @diegoreymendez as I don't understand all the moving parts here.

**Comment by diegoreymendez** [comment]
> A little bit more testing revealed that the attribute is added one time for every switch between visual and html mode.

This is most likely due to the code we have to load the extra attributes.  On the Aztec side of things.

@SergioEstevao - If that's the case it would be a good idea to add a few unit tests for this, when the fix is uploaded on Aztec's side.

**Comment by elibud** [comment]
👏 🎉 💯 Yes, it works, no duplication, everything I tested worked as expected 👍

**Comment by diegoreymendez** [comment]
@SergioEstevao - Can you retarget this to `try/integrate-aztec-b9`?

That's the branch we'll use for any pre-Beta 9 work we do.

The idea behind this change is to start merging into develop stable versions of Aztec from now on.

**Comment by diegoreymendez** [comment]
Ping @SergioEstevao ^

**Comment by SergioEstevao** [comment]
@diegoreymendez done!

**Comment by designsimply** [comment]
Tested and confirmed that img attributes width, height, and wp-image-XX class are set properly for newly added images:

img_1390
<sup>Tested with madefortesting.com running WP 4.9 and JP 5.2 hosted at Bluehost using iPhone 6S, iOS 10.3.3, WPiOS 8.2.0.20170731.</sup>

**Pull request merged**